samwise.js

```diff
@@ -3,30 +3,30 @@
  *
```

Comment:
SCRIPT-This block of code could benefit from some comments explaining the logic behind it.